File: MobileApp/gymly/lib/models/workout.dart
class Workout {
  final String subjectId;
  final int id;
  final DateTime created;
  final int durationInMinutes;
  final String programTitle;
  final String programDescription;
  final String programContent;

  Workout(
    this.subjectId,
    this.id,
    this.created,
    this.durationInMinutes,
    this.programTitle,
    this.programDescription,
    this.programContent,
  );

  factory Workout.fromJson(Map<String, dynamic> json) {
    return Workout(
      json['subjectId'] as String,
      json['id'] as int,
      DateTime.parse(json["createDate"]),
      json["durationInMinutes"] as int,
      json["programTitle"] as String,
      json["programDescription"] as String,
      json["programContent"] as String,
    );
  }
}

File: Backend/Microservices/UsersService/UsersService.Domain/Entities/Workout.cs
using Common.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UsersService.Domain.Entities;

public class Workout : AuditableBaseEntity
{
  public string SubjectId { get; set; } = default!;
  public int DurationInMinutes { get; set; } = default!;
  public string ProgramTitle { get; set; } = default!;
  public string ProgramDescription { get; set; } = default!;
  public string ProgramContent { get; set; } = default!;
}
